index.bs

```diff
@@ -6659,9 +6659,9 @@ to grow organically as needed.
      otherwise. If this throws an exception |e|, return [=a promise rejected with=] |e|.
   1. If |result| is a {{Promise}}, then return |result|.
   1. Return [=a promise resolved with=] undefined.
- 1. Let |cancelAlgorithmWrapper| be an algorithm that runs these steps:
-  1. Let |result| be the result of running |cancelAlgorithm|, if |cancelAlgorithm| was given, or
-     null otherwise. If this throws an exception |e|, return [=a promise rejected with=] |e|.
+ 1. Let |cancelAlgorithmWrapper| be an algorithm that runs these steps given a value |reason|:
```

Comment:
I took this phrasing from elsewhere in the spec, and the phrasing you suggest doesn't occur. Is there a reason to introduce a third phrasing for this? I say third because in addition to the phrasing I used, there is also "Let |xyzAlgorithm| be the following steps, taking a |foo| argument:"